### ISSUE TITLE
Feature request: Fetching of the last submitted solution

### ISSUE THREAD

**Issue by priyanshul-govil** [IEEE-NITK/CodePal#82]
It would be beneficial to fetch the last submitted solution in case we're working with a `FAILED` problem before having this extension.

**Comment by adityachirania** [comment]
Yeah @Janmansh  had a similar idea as well! We'll add this to our to do list! Thanks
